client/components/docs/docs.constants.js

```diff
@@ -1,200 +0,0 @@
-angular.module("managerApp")
```

Comment:
I prefer to have all constants in one file instead of duplicating them in 3 files.

Reply:
There are 2 ways of handling constants in the project and I quite liked having them in more than one file, it's easier to read 😄 . Anyway, I have to rework this after #805